INSTALLATION.md

```diff
@@ -1,218 +1,156 @@
+    1. Run the following command to set the key in the environment variable:
 
-  - Windows: `$env:GOOGLE_APPLICATION_CREDENTIALS="C:\Users\username\Downloads\service-account-file.json"`
+        - Linux/macOS: 
 
-  1.  Install the [Firebase CLI](https://firebase.google.com/docs/cli#install_the_firebase_cli).
-  2.  Copy the `firebase_options.dart` file as it will be modified.
-  3.  Run the following commands in the project directory of talawa mobile app:
+                export GOOGLE_APPLICATION_CREDENTIALS="/home/user/Downloads/service-account-file.json"
+
+        - Windows: 
+  
+                $env:GOOGLE_APPLICATION_CREDENTIALS="C:\Users\username\Downloads\service-account-file.json"
+
+  2.  Install the [Firebase CLI](https://firebase.google.com/docs/cli#install_the_firebase_cli).
+  3.  Copy the `firebase_options.dart` file as it will be modified.
+  4.  Run the following commands in the project directory of talawa mobile app:
 
       - `firebase login`
       - `dart pub global activate flutterfire_cli`
       - `flutterfire configure`
 
-  4.  Select the project you created in the Firebase console in step 2.
-  5.  Add iOS and android platforms to the project.
-  6.  Overwrite the `firebase_options.dart` file if asked so.
-  7.  Copy the keys to `.env` file, for how to set keys refer to `.env.sample` file.
-  8.  Undo the changes made to the `firebase_options.dart` file by pasting the old content from step 2.
-
-13. The command below will run the talawa-api server in development environment.
-
-        npm run dev
-
-14. To stop the server use this keybind in the terminal where the above command is executed :-
-
-        CTRL + C
-
-## Installation using docker
-
-Pre-requisites :-
-
-1. `Docker` installed on your system. Follow the setup guide on [docker docs](https://docs.docker.com/get-docker/).
-2. Free port `27017` on your system's localhost.
-
-**Note:** We are not listing every possible thing you need to know about docker in the following steps. We expect you to have minimum working knowledge with docker if you're using this approach. If you don't know anything about docker it's better to either learn about docker first or go with the steps mentioned in the section `Installation using standard method`. Also docker downloads a lot of large images, if you are short on storage or with slow internet connection prefer using the standard method.
+  5.  Select the project you created in the Firebase console in step 2.
+  6.  Add iOS and android platforms to the project.
+  7.  Overwrite the `firebase_options.dart` file if asked so.
+  8.  Copy the keys to `.env` file, for how to set keys refer to `.env.sample` file.
+  9.  Undo the changes made to the `firebase_options.dart` file by pasting the old content from step 2.
 
-Follow these steps to get the api running :-
 
-1.  Clone this repo to your local machine
+# Setting up the remaining environment variables
```

Comment:
Please give a brief explanation of why these are necessary. Or refer to the parameter section that was previously requested.